### PR TITLE
Implement navigation to profiles and chats from contact requests insi…

### LINKED ISSUE #19649: Implement navigation to profiles and chats from contact requests inside Activity Center
### Summary

* When someone opens the activity center, and views their contact requests, then pressing on the contact tag should open a profile and/or a 1-1 chat?
  * At the moment:
    * an accepted contact request will navigate to 1-1 chat when the contact tag is pressed
    * a pending contact request will not navigate anywhere when the contact tag is pressed
    * a declined contact request will not navigate anywhere when the contact tag is pressed
    * a contact acceptance notification will not navigate anywhere when the contact is pressed
  * Ideally:
    * an accepted contact request will navigate to 1-1 chat when the contact tag is pressed
    * a pending contact request will navigate to the contact profile contact tag is pressed
    * a declined contact request will navigate to the contact profile when the contact tag is pressed
    * a contact acceptance notification will navigate to the contact profile when the contact is pressed

### Example

https://github.com/status-im/status-mobile/assets/2845768/416cfe94-e494-4a5e-bdf3-cc8057bfd553

### PR DESCRIPTION
**Pull request by flexsurfer**
fixes #19649

### CODE REVIEW

File: src/status_im/contexts/shell/activity_center/notification/contact_requests/view.cljs [hunk 1]
```diff
@@ -165,4 +165,8 @@
         [incoming-contact-request-view props]]
 
        :else
-       [incoming-contact-request-view props])]))
+       [gesture/touchable-without-feedback
+        {:on-press (fn []
+                     (rf/dispatch [:hide-popover])
```
Comment (ibrkhalil) [inline src/status_im/contexts/shell/activity_center/notification/contact_requests/view.cljs:170]: nit: Why are we using :hide-popover in this namespace, Shouldn't the convention be `:navigate-back`?
CC: @J-Son89 @Parveshdhull

**Review by ibrkhalil — APPROVED** [review]
lgtm, With a single nit

File: src/status_im/contexts/shell/activity_center/notification/contact_requests/view.cljs (hunk 1 above)
Comment (J-Son89) [reply src/status_im/contexts/shell/activity_center/notification/contact_requests/view.cljs:170]: @briansztamfater @flexsurfer might know

Comment (ibrkhalil) [reply src/status_im/contexts/shell/activity_center/notification/contact_requests/view.cljs:170]: I think `:navigate-back` should handle the popover if it's open
maybe it should dispatch `:hide-popover` if there's an open popover

Comment (Parveshdhull) [reply src/status_im/contexts/shell/activity_center/notification/contact_requests/view.cljs:170]: I think activity center is a modal, so `:navigate-back` should work for closing it. (or we are using :hide-popover for some other purpose?)

Comment (flexsurfer) [reply src/status_im/contexts/shell/activity_center/notification/contact_requests/view.cljs:170]: nobody knows :)

**Comment by mariia-skrypnyk** [comment]
Hi @flexsurfer !

I am moving back your PR to the corresponding column as it doesn't meet the requirements outlined here:
https://github.com/status-im/status-mobile/blob/develop/doc/pipeline_process.md

Please, make sure your fix should't be tested by QA's and if so - add necessary information to PR's description.
Thanks!

**Pull request closed without merging**